### PR TITLE
Feat: List toast's component props fs-632

### PR DESCRIPTION
**Pull request by eduardoformiga**
## What's the purpose of this pull request?

Adds `PropsSection` to Toast component.

As the `Toast` haven't have Props, since the toast is triggered by `pushToast()` event, I also added this instruction.
Screenshot 2023-03-22 at 13 10 30

## How do we test?
Check the preview https://faststore-site-hogi92pkm-faststore.vercel.app/

### CODE REVIEW

File: apps/site/pages/components/molecules/toast.mdx [hunk 1]
```diff
@@ -200,7 +200,34 @@ import '@faststore/ui/src/components/molecules/Toast/styles.scss'
...
+  {
+    name: 'status',
+    type: "'ERROR' | 'WARNING' | 'INFO'",
+    description: "Toast's status type.",
+    required: true,
+  },
+  {
+    name: 'icon',
+    type: 'ReactNode',
+    description: "Toast's icon.",
```
Comment (hellofanny) [inline apps/site/pages/components/molecules/toast.mdx:224]:
```suggestion
    description: "A React component that will be rendered as toast's icon.",
```

?

File: apps/site/pages/components/molecules/toast.mdx [hunk 2]
```diff
@@ -200,7 +200,34 @@ import '@faststore/ui/src/components/molecules/Toast/styles.scss'
...
+  },
+]
+
+<p style={{marginBottom: "2rem"}}>Toast's props are in the object passed as argument by `pushToast(toast: Props)` method.</p>
```
Comment (hellofanny) [inline apps/site/pages/components/molecules/toast.mdx:228]:
```suggestion
<p style={{marginBottom: "2rem"}}>Toast's props are in the object passed as an argument by `pushToast(toast: Props)` method.</p>
```

File: apps/site/pages/components/molecules/toast.mdx [hunk 3]
```diff
@@ -200,7 +200,34 @@ import '@faststore/ui/src/components/molecules/Toast/styles.scss'
...
+  },
+  {
+    name: 'message',
+    type: 'string',
+    description: 'Message shown in Toast.',
+    required: true,
+  },
+  {
+    name: 'status',
+    type: "'ERROR' | 'WARNING' | 'INFO'",
```
Comment (renatamottam) [inline apps/site/pages/components/molecules/toast.mdx:217]: does it need to be all caps?

**Review by renatamottam — APPROVED** [review]

File: apps/site/pages/components/molecules/toast.mdx (hunk 3 above)
Comment (hellofanny) [reply apps/site/pages/components/molecules/toast.mdx:217]: I think so, it's how we defined before.

Comment (eduardoformiga) [reply apps/site/pages/components/molecules/toast.mdx:217]: yes, that's the way it's defined in the `UIProvided`.

**Pull request merged**